JWBestPracticeApps/BasicPlayer/BasicPlayerViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  BasicPlayer
+//
+//  Created by David Perez on 19/11/20.
+//  Copyright © 2020 JWPlayer. All rights reserved.
+
+import UIKit
+import JWPlayerKit
+
+/**
+ Provides minimal setup for a JWPlayerViewController.
+ Create an instace of this player if you need a simple JWPlayerView.
+*/
+class BasicPlayerViewController: JWPlayerViewController {
+    private let videoUrlString = "https://playertest.longtailvideo.com/adaptive/oceans/oceans.m3u8"
+    private let posterUrlString = "https://d3el35u4qe4frz.cloudfront.net/bkaovAYt-480.jpg"
```

Comment:
It is just a visual thing, but the poster image doesn't match the video content for both (swift and obj-c)

Reply:
Thank you for this, I will address this in the BasicPlayer ticket.